### PR TITLE
Update key repositories guide

### PR DESCRIPTION
**Pull request by denniskigen**
Wholesale updates to the key repositories guide.

### CODE REVIEW

File: pages/docs/key-repositories.en-US.mdx [hunk 1]
```diff
@@ -1,83 +1,85 @@
...
+
+These repositories are domain-specific, meaning that the code in each repository is typically focused on a particular concern. For example, the `Core` repository contains packages that handle cross-cutting concerns such as loading frontend modules, configuration, internationalization, rendering the UI, routing, error handling, styling, and so forth. The `Patient Chart` repository contains frontend modules that constitute widgets in a patient dashboard. Some of these repositories are structured as monorepos, meaning that they contain multiple packages, while others are more traditional, containing a single package. In cases where a repository contains multiple packages, it is usually because those packages share a single concern or theme. For example, the `Patient Chart` repository aggregates packages that handle concerns around a typical patient dashboard, including views for allergies, appointments, conditions, forms, medications, notes, programs, test results, vitals, and more.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:5]:
```suggestion
The code in each repository is focused on a particular concern. For example, the [Patient Chart](https://github.com/openmrs/openmrs-esm-patient-chart) repository contains frontend modules for the features of the patient chart, such as allergies, appointments, conditions, forms, medications, notes, programs, test results, and vitals. This is an example of a monorepo, where multiple packages are contained in a single repository; most O3 repositories are structured in this way. The [Core](https://github.com/openmrs/openmrs-esm-core) repository is special. It contains a few frontend modules, but also the core application logic that loads  frontend modules, as well as the [O3 tooling](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/tooling/openmrs) and the [JavaScript API](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md).
```

File: pages/docs/key-repositories.en-US.mdx [hunk 2]
```diff
@@ -1,83 +1,85 @@
...
-- [Frontend RFC](#frontend-rfc) - where architectural decisions affecting technologies and processes that apply to all new O3 distributions are first presented, argued, and finally agreed upon via consensus.
-- [JSON schemas](#json-schemas) - where standard JSON schemas used in OpenMRS are hosted.
-
-## Overview
-
-Our major repositories are set up as monorepos. A monorepo is a collection of many different applications and packages in a single codebase. Typically, these monorepos are set up using [`lerna` and `yarn` workspaces](https://medium.com/@jsilvax/a-workflow-guide-for-lerna-with-yarn-workspaces-60f97481149d). The monorepo will have one root-level `package.json` manifest file which contains all the root-level scripts in addition to the `package.json` files of all of its constituent packages.
+- [Core](#core) - aggregates the core frontend modules of the O3 frontend that handle essential cross-cutting concerns such as module loading, configuration, internationalization, UI rendering, feature flagging, routing, global state management, and more.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:9]:
```suggestion
- [Core](#core) - contains frontend modules for low-level concerns such as authentication. It also contains the base application which loads and stitches together all frontend modules, the app shell. This app shell provides the JavaScript library `@openmrs/esm-framework`, which provides for essential cross-cutting concerns such as configuration, internationalization, feature flagging, and global state management. This repository also contains the tooling for O3.
```

File: pages/docs/key-repositories.en-US.mdx [hunk 3]
```diff
@@ -1,83 +1,85 @@
...
+- [Module management](#module-management) - a learning frontend module based on the legacy OpenMRS Module management UI. The code in this repo is annotated with comments and can serve as a good starting point for new developers looking to understand how to build frontend modules.
+- [Frontend RFC](#frontend-rfc) - a repository where architectural decisions affecting technologies and processes that apply to all new O3 distributions are first presented, argued, and finally agreed upon via consensus.
+- [JSON schemas](#json-schemas) - a repository that hosts [standard JSON schemas](https://json-schema.org/) used in OpenMRS. These schemas are used to provide auto-completion and validation capabilities when editing routes registry files for O3 frontend modules and O3 form schemas.
 
 ### Core
 
-Most of the time, you'll likely be working with either the `patient chart` or `patient management` repositories. However, you might find yourself occasionally needing to delve into `core`. The packages available in core include:
+Core aggregates a mix of core apps that include both typical frontend modules as well as packages that don't render UIs. Typical frontend modules include:
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:29]:
```suggestion
Core contains both typical frontend modules as well as packages that don't render UIs. Typical frontend modules include:
```

File: pages/docs/key-repositories.en-US.mdx [hunk 4]
```diff
@@ -1,83 +1,85 @@
...
-- [Shell](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/shell) - the app shell, which forms the base of the application, and is tasked with doing a [whole host of things](/docs/core-concepts#app-shell), from registering frontend modules in the import map to starting the application.
-- [Tooling](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/tooling) - the [openmrs CLI](http://npm.im/openmrs) which is the one-stop CLI for O3 that's useful for firing dev servers, assembling frontend modules for distribution, and more.
+- [Core framework](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/framework) - aggregates packages that make up the core O3 framework library, including shared API objects (such as the patient and visit objects), the breadcrumbs menu, the extension system, shared React utils, global state management tools, error handling, and the style guide, among other things. The functionality exported by these packages gets aggregated into an NPM JavaScript package called [esm-framework](http://npm.im/@openmrs/esm-framework) that gets reused by most frontend modules.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:39]:
```suggestion
- [Core framework](https://github.com/openmrs/openmrs-esm-core/tree/main/packages/framework) - aggregates packages that make up the core [O3 framework API](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md), including shared API objects (such as the patient and visit objects), the breadcrumbs menu, the extension system, shared React utils, global state management tools, error handling, and the style guide, among other things. The functionality exported by these packages gets aggregated into an NPM JavaScript package called [esm-framework](http://npm.im/@openmrs/esm-framework) that gets reused by most frontend modules.
```

File: pages/docs/key-repositories.en-US.mdx [hunk 5]
```diff
@@ -106,21 +109,22 @@ The [Angular form engine](https://github.com/openmrs/openmrs-ngx-formentry) is a
...
+
+The [React form engine](https://github.com/openmrs/openmrs-form-engine-lib) is a React library that enables the creation and rendering of standardized medical forms within the O3 ecosystem. It provides a flexible, JSON-based form schema system with built-in validation, conditional rendering, and multilingual support - making it ideal for complex healthcare data collection needs. O3 form schemas are written in JSON and conform to the [O3 standard JSON schema spec](https://github.com/openmrs/openmrs-contrib-json-schemas/blob/main/form.schema.json) which defines the structure and constraints of the form data. The [O3 reference application](https://o3.openmrs.org/openmrs/spa) ships an embedded [Form Builder](https://github.com/openmrs/openmrs-esm-form-builder) that users can leverage to build forms interactively, using the interactive schema builder, or by writing code into the built-in JSON schema editor. Learn more about the React form engine from the [official docs](https://openmrs.atlassian.net/wiki/spaces/projects/pages/68747273/O3+Form+Docs).
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:113]:
```suggestion
The [React form engine](https://github.com/openmrs/openmrs-form-engine-lib) is a library that enables creating and rendering medical forms within O3. It provides a flexible, JSON-based form schema with built-in validation, conditional rendering, and multilingual support. O3 form schemas conform to the [O3 standard JSON schema spec](https://github.com/openmrs/openmrs-contrib-json-schemas/blob/main/form.schema.json) which defines the structure and constraints of the form data.   Learn more about the React form engine from the [official docs](https://openmrs.atlassian.net/wiki/spaces/projects/pages/68747273/O3+Form+Docs). The [O3 reference application](https://o3.openmrs.org/openmrs/spa) ships an embedded [Form Builder](https://github.com/openmrs/openmrs-esm-form-builder) to make building forms easy.
```

...and then the reader reads on to the next section.

BTW, my preference would be to capitalize things like React Form Engine and Form Builder as proper nouns.

File: pages/docs/key-repositories.en-US.mdx [hunk 6]
```diff
@@ -106,21 +109,22 @@ The [Angular form engine](https://github.com/openmrs/openmrs-ngx-formentry) is a
...
+The [Form builder](https://github.com/openmrs/openmrs-esm-form-builder) is a frontend module used to create OpenMRS form schemas. It enables users to both create new schemas and edit existing ones. It provides an embedded code editor that accepts JSON code and an interactive editor where users can construct a schema interactively without writing code. Schemas built using the form builder can be persisted to the server, and optionally published, which makes them available to the frontend via the forms widget. Schemas built in the form builder get rendered via the React form engine, which allows users to validate and test their forms straight from the form builder UI.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:117]:
```suggestion
The [Form builder](https://github.com/openmrs/openmrs-esm-form-builder) is a frontend module used to create OpenMRS form schemas, for use with the React form engine. It enables users to both create new schemas and edit existing ones. It provides an embedded code editor that facilitates writing JSON code, and a WYSIWYG schema builder interface where users can construct a schema interactively without writing code. Schemas built using the form builder can be persisted to the server, and optionally published, which makes them available to the frontend via the forms widget. The React form engine is used in the form builder to validate and test forms directly in the form builder UI.
```

File: pages/docs/key-repositories.en-US.mdx [hunk 7]
```diff
@@ -106,21 +109,22 @@ The [Angular form engine](https://github.com/openmrs/openmrs-ngx-formentry) is a
...
 
 ### Fast data entry
 
-The [Fast Data Entry App](https://github.com/openmrs/openmrs-esm-fast-data-entry-app) is a frontend module that allows for a natural workflow when entering many pre-recorded forms at a time. It is not meant for point-of-care workflows, but rather as a way to do retrospective data entry. It allows the rapid input of forms for group sessions enabling recording information about a group session or for session participants.
+The [Fast data entry app](https://github.com/openmrs/openmrs-esm-fast-data-entry-app) is a frontend module that allows for a natural workflow when entering many pre-recorded forms at a time. It is not meant for point-of-care workflows, but rather as a way to do retrospective data entry. It allows the rapid input of forms for group sessions enabling recording information about a group session or for session participants.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:121]:
```suggestion
The [Fast data entry app](https://github.com/openmrs/openmrs-esm-fast-data-entry-app) is a frontend module that provides a natural workflow for entering many pre-recorded forms at a time. It is not meant for point-of-care workflows, but rather as a way to do retrospective data entry. It also can be used for the rapid input of forms for group sessions.
```

File: pages/docs/key-repositories.en-US.mdx [hunk 8]
```diff
@@ -106,21 +109,22 @@ The [Angular form engine](https://github.com/openmrs/openmrs-ngx-formentry) is a
...
+The [Medication dispensing app](https://github.com/openmrs/openmrs-esm-dispensing-app) is a simple tool that enables tracking dispensed medications. It's designed to serve small sites that don't want to deal with implementing a whole other external supply chain/ERP system. These sites typically only care about tracking what needs to be dispensed vs what has already been dispensed. The primary target users for this functionality are implementations and facilities that have an on-site pharmacy and don't charge the patient/customer for the services, who want to dispense medications based on orders within the same OpenMRS instance. Billing and inventory are out of scope.
```
Comment (brandones) [inline pages/docs/key-repositories.en-US.mdx:125]:
```suggestion
The [Medication dispensing app](https://github.com/openmrs/openmrs-esm-dispensing-app) is a simple tool for tracking dispensed medications. It's designed to serve small sites that don't want to deal with implementing a whole external supply chain/ERP system. It simply tracks what needs to be dispensed and what has already been dispensed. The primary target users for this functionality are implementations and facilities that have an on-site pharmacy and don't charge the patient/customer for the services, and who want to dispense medications based on orders within the same OpenMRS instance. Billing and inventory are out of scope.
```

File: pages/docs/key-repositories.fr-FR.mdx [hunk 9]
```diff
@@ -1,141 +1,161 @@
...
-- [JSON schemas](#json-schemas) - où sont hébergés les schémas JSON standards utilisés dans OpenMRS.
-
-## Vue d'ensemble [#overview]
-
-Nos principaux dépôts sont configurés comme des monorepos. Une monorepo est une collection de plusieurs applications et paquets différents dans une base de code unique. Typiquement, ces monorepos sont mis en place en utilisant [`lerna` et `yarn` workspaces](https://medium.com/@jsilvax/a-workflow-guide-for-lerna-with-yarn-workspaces-60f97481149d). La monorepo aura un fichier manifeste `package.json` de niveau racine qui contient tous les scripts de niveau racine en plus des fichiers `package.json` de tous les paquets qui la composent.
+# Dépôts clés
+
+Le code d'O3 est hébergé dans plusieurs dépôts GitHub maintenus par la communauté. O3 utilise une [architecture basée sur les microfrontends](https://microfrontends.org) - une approche où l'application est divisée en applications frontales plus petites et indépendantes qui fonctionnent ensemble - ce qui signifie que le code des différentes parties de l'application est réparti sur plusieurs dépôts.
```
Comment (brandones) [inline pages/docs/key-repositories.fr-FR.mdx:3]: Malhueresment je ne peux pas reviser ça

**Review by brandones — APPROVED** [review]
Thanks for doing all this! This is really good & useful content, solid improvements. I did a bunch of editing, mostly just simplifying, but also a little rearranging and clarifying around Core. Approving so that you can merge once you incorporate edits as desired.

**Review by brandones — APPROVED** [review]

**Review by NethmiRodrigo — APPROVED** [review]

**Pull request merged**